### PR TITLE
#100 azure terraform with docker

### PR DESCRIPTION
**Pull request by crabulik**
A multi region solution for deploying the db1000n container on Azure Container Group

### CODE REVIEW

File: terraform/azure/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    - bomblet_count=10   - can be used for custom number of containers per region.
+
+- ```terrafrom init``` - to restore all dependencies.
+
+- ```terraform apply -auto-approve``` - to provision the attack farm. 
+
+
+## Collecting logs from the containers
+
+The container instances are provisioned without public IP adresses to make the setup more cost effective. If you deploy more than one container per region, play with the -01 sufix to get logs from the correct instance.
```
Comment (Amet13) [inline terraform/azure/README.md:24]:
```suggestion
The container instances are provisioned without public IP addresses to make the setup more cost-effective. If you deploy more than one container per region, play with the `-01` suffix to get logs from the correct instance.
```

File: terraform/azure/bomblet/main.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+locals {
+
+}
```
Comment (Amet13) [inline terraform/azure/bomblet/main.tf:3]:
```suggestion
locals {}
```

File: terraform/azure/main.tf [hunk 3]
```diff
@@ -0,0 +1,63 @@
+
```
Comment (Amet13) [inline terraform/azure/main.tf:1]:
```suggestion
```

File: terraform/azure/bomblet/variables.tf [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  default = "1"
+}
+
+variable "attack_memory" {
+  default = "1.5"
+}
+
+variable "attack_environment_variables" {
+  default = {
+    }
```
Comment (Amet13) [inline terraform/azure/bomblet/variables.tf:27]:
```suggestion
  }
```

File: terraform/azure/variables.tf [hunk 5]
```diff
@@ -0,0 +1,11 @@
+variable "bomblet_count" {
+  type    = number
+  default = 1
+  description = "Number of containers per region."
+}
+
```
Comment (Amet13) [inline terraform/azure/variables.tf:6]:
```suggestion
```

**Review by Amet13 — CHANGES_REQUESTED** [review]
Hey.
Could you please solve mentioned issues?
Also please remove empty `outputs.tf` files and run terraform fmt to format the code?

**Comment by crabulik** [comment]
@Amet13 Done

**Review by Amet13 — APPROVED** [review]

**Pull request merged**